### DISCUSSION TITLE
Limits with OPDS Support due to authentication

### DISCUSSION THREAD

**Discussion by kulak** [stumpapp/stump#515] [General]
Multiple mobile apps support OPDS *without* authentication.

I tested 3 apps on iOS and only one provided user and password (Yomu):

- [Yomu Reader](https://apps.apple.com/us/app/yomu-ebook-reader/id562211012) provides user id and password and as a result works.
- [KyBook 3](https://apps.apple.com/us/app/kybook-3-ebook-reader/id1348198785) does not provide user id and password.  KyBook 3 still works on iOS as long as access to local network is enabled manually in settings -> apps -> KyBook 3.
- [Cantook by Aldiko](https://apps.apple.com/us/app/cantook-by-aldiko/id1476410111) does not take user id and password.  I liked this one the most.

I used [dir2opds](https://github.com/dubyte/dir2opds) to test apps without user id and password.

**Comment by aaronleopold** [comment]
You can use an API key to get around the credentials requirement: https://www.stumpapp.dev/guides/opds#using-an-api-key
